spec/features/04_user_deletes_account_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature "user deletes their account" do
+  scenario "user wants to delete their account" do
+    visit '/'
+    click_on 'Sign In'
+    click_on 'Sign up'
+    fill_in 'Username', with: Faker::Internet.user_name
+    fill_in 'Email', with: Faker::Internet.email
+    fill_in 'Password', with: 'hello1234TEST'
+    fill_in 'Password confirmation', with: 'hello1234TEST'
+    click_on 'Sign up'
```

Comment:
You can just delete lines 5-12 and replace them with creating a user and signing them in.